### ISSUE TITLE
Don't cause an exception in _pfp__children

### ISSUE THREAD

**Issue by carlosgprado** [d0c-s4vage/pfp#32]
I don't know if this would break something but wouldn't be better to have every element default to have an attribute __pfp__children_ equal to an empty list?

This way when write something like `obj._pfp__children` in say, traversing the DOM's graph (DFS or alike) you don't have to worry about exceptions and the code is more succinct. 

Right now I am using an auxiliary function like:

``` python
def has_children(n):
    """ Returns boolean """
    try:
        n._pfp__children
        return True

    except AttributeError:
        return False
```

but it looks cumbersome and feels more natural to return an empty list to the question "give me a list containing your children"

Anyway, awesome work man :+1:

**Comment by d0c-s4vage** [comment]
Good point. Man I need to get around to fixing the other issues. Didn't realize so many have piled up..
